bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala

```diff
@@ -137,10 +137,9 @@ final class TpeGhostIfuFeature extends TpeImageFeature("GHOST", "Show the patrol
 
               // IFU2
               val p2 = pm.getLocationFromTag(s)
-              // We don't draw the SRIFU2, apparently, even though its position
-              // determines the position of the HR sky fibers
-              // drawStandardResolutionIfu(g2d, p2, Ifu2, AsSky)
-              drawHighResolutionSky(g2d, p2)
+              // We draw the SRIFU2 as Sky
+              drawStandardResolutionIfu(g2d, p2, Ifu2, AsSky)
+              //drawHighResolutionSky(g2d, p2)
```

Comment:
I was a bit unsure whether to leave this on but it seems odd to draw two markers at the same position

Reply:
Do the appear at the same place?  I thought the HR sky fibers were slightly offset.  At any rate, we can ask for feedback with this version.

Reply:
Yes they show in the same place:
![image](https://user-images.githubusercontent.com/3615303/199498392-6e3f38bd-e41d-4785-be3e-f24fae010ac1.png)